core/src/serialize.rs

```diff
@@ -47,6 +46,25 @@ impl fmt::Display for ExportFormat {
     }
 }
 
+/// Available common export formats.
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Default, clap::ValueEnum)]
+pub enum ExportFormatCommon {
+    #[default]
+    Json,
```

Comment:
I wonder if we shouldn't rather have an additional `Markdown` variant, which is the default (and should be the documented default in the CLI as well). I think it's good practice that the normal behavior (without `--format`) can be recovered in a more precise way with an argument - typically, `nickel export` is the same as `nickel export --format json`, etc. - so that the default behavior is just that, a default value.

On the other hand, it makes the logic of when we should allow `--doc`, etc. more complex. So I'm not entirely sure. Maybe we can start as it is now, but then I would remove the `Default` impl of `ExportFormatCommon`, because it doesn't really have a default value.